plugins/modules/win_domain_group_membership.ps1

```diff
@@ -50,7 +50,11 @@ if ($diff_mode) {
     $result.diff = @{}
 }
 
-$members_before = Get-AdGroupMember -Identity $ADGroup @extra_args
+$group = Get-ADGroup $ADGroup @extra_args
+$filter = " (&(objectClass=groupOfNames)(memberOf=$($group.DistinguishedName)))"
```

Comment:
Can you explain the `objectClass=groupOfNames` part? When using this filter I get nothing returned back to me on some of my AD groups.

There's also a leading space here that probably isn't needed.

Reply:
...good catch: it doesn't make sense. I propose to remove the class filter as we can have groups of any ldap object class.